disable-libx11-xcb-support.patch
origin: http://gitweb.opencompositing.org/?p=users/3v1n0/compiz-patches;a=blob_plain;f=disable-libx11-xcb-support.patch;hb=4b8e083acf9b1779a9de43ed295b6a4a86594f10
locally modified to reflect 0.7.4 updates
--- 0-compiz.orig/configure.ac
+++ 0-compiz/configure.ac
@@ -118,8 +118,7 @@ AC_SUBST(imagedir)
 metadatadir=$datadir/compiz
 AC_SUBST(metadatadir)
 
-COMPIZ_REQUIRES="x11-xcb    \
-		 xcomposite \
+COMPIZ_REQUIRES="xcomposite \
 		 xfixes	    \
 		 xdamage    \
 		 xrandr	    \
--- 0-compiz.orig/include/compiz-core.h
+++ 0-compiz/include/compiz-core.h
@@ -33,7 +33,7 @@
 #include <stdio.h>
 #include <sys/time.h>
 
-#include <X11/Xlib-xcb.h>
+#include <X11/Xlib.h>
 #include <X11/Xutil.h>
 #include <X11/extensions/Xdamage.h>
 #include <X11/extensions/Xcomposite.h>
@@ -897,8 +897,6 @@ struct _CompDisplay {
 
     CompDisplay *next;
 
-    xcb_connection_t *connection;
-
     Display    *display;
     CompScreen *screens;
 
--- 0-compiz.orig/src/display.c
+++ 0-compiz/src/display.c
@@ -2054,8 +2054,6 @@ addDisplay (const char *name)
 	return FALSE;
     }
 
-    d->connection = XGetXCBConnection (dpy);
-
     if (!compInitDisplayOptionsFromMetadata (d,
 					     &coreMetadata,
 					     coreDisplayOptionInfo,
